app/templates/suppliers/_frameworks_coming.html

```diff
@@ -1,19 +1,3 @@
 {% for framework in frameworks.coming %}
-  {% if framework.slug == 'digital-outcomes-and-specialists' %}
-
-    {% set blog_link %}
```

Comment:
`blog_link`! I thought we removed all these 😹 Good spot

Reply:
I couldn't believe my eyes 😂